settings.gradle

```diff
@@ -20,3 +20,7 @@ include ':stripecardscan'
 include ':stripecardscan-example'
 include ':stripecardscan-tflite'
 include ':wechatpay'
+include ':ml-core-base'
```

Comment:
Note it's not possible to declare this in a single modules with different flavors, please see details [here](https://docs.google.com/document/d/1l4cmSbvimpLq0cZab09i4WYjYbAmRew814V1_SD9hNo/edit#bookmark=id.rxcjd2rk2sdw)

Reply:
An alternative could be to nest these all under `ml-core/base`, `ml-core/default`, etc. But it would still end up being multiple gradle projects, and require multiple lines in the settings gradle file.

Reply:
will leave them here as is they are technically different modules